includes/admin/class-wp-job-manager-writepanels.php

```diff
@@ -556,6 +556,20 @@ public function save_job_listing_data( $post_id, $post ) {
 				}
 			}
 		}
+
+		/* Set Post Status To Expired If Already Expired */
+		$expiry_date = get_post_meta( $post_id, '_job_expires', true );
+		$today_date  = date( 'Y-m-d', current_time( 'timestamp' ) );
+		$post_status = $expiry_date && $today_date > $expiry_date ? 'expired' : false;
+		if( $post_status ){
```

Comment:
Mind [adding space around parenthesis](https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#space-usage) of this `if ( ... ) {}` statement?